### PR TITLE
Feat/terms and conditions

### PR DESCRIPTION
**Pull request by anyulled**
Adding terms and conditions page

[Preview](https://deploy-preview-3--dev-bcn.netlify.app/conditions)

### CODE REVIEW

File: public/CNAME [hunk 1]
```diff
@@ -0,0 +1 @@
+www.devbcn.com
```
Comment (icougil) [inline public/CNAME:1]: Why we have to remove this?

Comment (anyulled) [reply public/CNAME:1]: It's the other way around, I'm adding it to this folder, to ensure it's always present on the `gh-pages` branch.

Comment (icougil) [reply public/CNAME:1]: ah, got it. But if it was not included before... why you need to add it now? I mean, the web is working right now with the CNAME in the main/master branch...

Comment (anyulled) [reply public/CNAME:1]: yes, indeed, it's already on the main branch, but not on the `gh-pages` and I think that's the reason every time we publish the web the domain is reset. Adding it to the public folder will include it as part of the build files.

**Review by jonathanvila — CHANGES_REQUESTED** [review]
Yo cambiaria el mail "info@devbcn.com" por el mail que designemos para los clientes ( tickets@devbcn.com o customers@devbcn.com )

**Comment by icougil** [comment]
Agree, al final se ha quedado [customers@devbcn.com](mailto:customers@devbcn.com) )

Btw, que comentaba en Slack, deberíamos incluir lo que tenemos de Conversia...

**Comment by anyulled** [comment]
> Agree, al final se ha quedado [customers@devbcn.com](mailto:customers@devbcn.com) )
> 
> Btw, que comentaba en Slack, deberíamos incluir lo que tenemos de Conversia... 
> 
> 

I know...

**Pull request merged**